openapi/schemas/processes-core/statusInfo.yaml

```diff
@@ -11,12 +11,24 @@ allOf:
       processID:
         type: string
         format: uri
-      type:
-        type: string
-        example:
-          - process
-          - wps
-          - openeo
+      processingEntityType:
```

Comment:
`type` is still marked `required` but doesn't exist anymore. It should be replaced by `processingEntityType`

Reply:
Done!